docs/src/pages/docs/crocks/Const.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Const"
+description: "Const Monoid"
+layout: "guide"
+weight: 80
+---
+
+```haskell
+Const a
+```
+
+`Const` is a `chainable` that will take a any value `a` and regardless of the function called return a new `Const` with the same value.
+
+```javascript
+import Const from 'crocks/Const'
+
+Const('Hello World')
+//=> Const 'Hello World'
+
+([ 'Today', 'Tomorrow', 'Yesterday' ])
+   .map(Const)
+   .reduce ((acc, c) => acc.concat(c))
+//=> Const 'Today'
+
+Const(100)
+  .concat(Const(10))
+//=> Min 100
+
+Const('Hello')
+  .map(x => x + x)
+//=> Const 'Hello'
+```
+
+<article id="topic-implements">
+
+## Implements
+
+`ap`, `chain`, `concat`, `equals`, `map`
+</article>
+
+<article id="topic-instance">
+
+## Instance Methods
+
+#### concat
+
+```haskell
+Min ~> Min -> Min
+```
+
+`concat` is used to combine (2) `Semigroup`s of the same type under an
+operation specified by the `Semigroup`. In the case of `Min`, it will result
```

Comment:
Here, `Min` is introduced for the first time as an aside. "In the case of `Min`"... As the reader, I didn't know `Min` was a topic until it already became a case :D Perhaps a brief introduction is in order, since the document discusses `Const`, whereas this whole section doesn't mention `Const` at all.